motor_controller:
  ros__parameters:
    wheel_base: 1.0
    wheel_diameter: 0.2372
    gear_ratio: 40
    max_rpm: 1500
    

